BinaryEncoding.md

```diff
@@ -335,9 +332,9 @@ The `br_table` operator has an immediate operand which is encoded as follows:
 
 | Field | Type | Description |
 | ---- | ---- | ---- |
-| target_count | `uint16` | number of targets in the target_table |
-| target_table | `uint16*` | target entries that indicate an outer block or loop to which to break |
-| default_target | `uint16` | an outer block or loop to which to break in the default case |
+| target_count | `varuint32` | number of targets in the target_table |
+| target_table | `varuint32*` | target entries that indicate an outer block or loop to which to break |
```

Comment:
Hah, bummer. How about this: use leb128, but with the extra constraint that all values in the table must use an encoding with the same width (using leading zero as necessary)? That way, an interpreter would only have to look at the first index to know the size of the table.